src/cvode/cvode_nls.c

```diff
@@ -284,13 +295,13 @@ static int cvNlsLSetup(booleantype jbad, booleantype* jcur, void* cvode_mem)
 }
 
 
-static int cvNlsLSolve(N_Vector delta, void* cvode_mem)
+static SUNNlsStatus cvNlsLSolve(N_Vector delta, void* cvode_mem)
```

Comment:
This routine (and others in this file) are inconsistent in their use of `SUNNlsStatus`.  For example, this routine assumes that `CV_MEM_NULL` and `CV_LSOLVE_FAIL` are valid `SUNNlsStatus` values.